.github/workflows/ipc_fuzzer.yml

```diff
@@ -57,10 +57,24 @@ jobs:
           sudo dpkg --add-architecture i386
           sudo apt-get update
 
-      - name: apt-get install
-        run: sudo apt-get update && sudo apt-get -y install
-          clang llvm libc6-dev-i386 libstdc++-12-dev:i386
-          ninja-build device-tree-compiler python3-pyelftools
+      - name: apt-get install i386 packages
+        run: |
+
+          # We don't know why Github requires us to explicit all these
+          # i386 dependencies. On a vanilla Ubuntu 22.04 the last
+          # apt-get is enough.
+
+          sudo apt-get -y install libgcc-s1:i386 libc6-dev:i386 libstdc++6:i386
+          sudo apt-get -y install libgomp1:i386  libitm1:i386 libatomic1:i386
+          sudo apt-get -y install libasan8:i386 libubsan1:i386 libquadmath0:i386
+
+          sudo apt-get -y install \
+             libc6-dev-i386 libc6-dev:i386 libstdc++-12-dev:i386
```

Comment:
Nitpick: combining all the apt-get packages into a single command avoids having to get the ordering right manually and can sometimes run noticeably faster (if the graph is deep, probably not here) by avoiding the extra work to traverse it multiple times.

Reply:
Half-done :-) I reduced the number of `apt install` but kept a separate, non-Github specific command that users of "normal" Ubuntu 22.04 can re-use as is.

I also kept the non-i386 packages separate in case i386 gets broken again somehow.